spec/services/processors/blog_technology_views_updater_spec.rb

```diff
@@ -87,35 +89,7 @@
       end
 
       it 'creates a technology metric for each one of those months' do
-        expect { described_class.call }.to change { Metric.count }.by 2
-      end
-
-      context 'but technologies visits metrics have already been registered' do
-        before do
-          create(
-            :metric,
-            name: Metric.names[:blog_visits],
-            interval: Metric.intervals[:monthly],
-            value: last_month_blog_post_1_views_metric.value,
-            value_timestamp: metric_timestamp.last_month,
-            ownable: technology
-          )
-
-          create(
-            :metric,
-            name: Metric.names[:blog_visits],
-            interval: Metric.intervals[:monthly],
-            value: technology_visits,
-            value_timestamp: metric_timestamp,
-            ownable: technology
-          )
-        end
-
-        it 'just tries to update the last of them' do
```

Comment:
why is this test removed?

Reply:
It was moved actually, to `blog_technology_views_partial_updater_spec.rb` (https://github.com/rootstrap/rs-code-review-metrics/pull/196/files#diff-ce642b8ab10a78e0f9b5844865b268bbR48)
This behaviour is not related anymore to this class, it depends on the subclass being used (`BlogTechnologyViewsPartialUpdater ` or `BlogTechnologyViewsFullUpdater `)